libknet/libknet.h

```diff
@@ -576,6 +577,23 @@ int knet_handle_enable_filter(knet_handle_t knet_h,
 
 int knet_handle_setfwd(knet_handle_t knet_h, unsigned int enabled);
 
+/**
+ * knet_handle_setprio_dscp
+ *
+ * @brief Use dscp for IP_TOS on socket to implement KNET_LINK_FLAG_TRAFFICHIPRIO
+ *
+ * knet_h   - pointer to knet_handle_t
+ *
+ * dscp     - dscp value to set on all new sockets
+ *
+ * @return
+ * knet_handle_setfwd returns
+ * 0 on success
+ * -1 on error and errno is set.
+ */
+
+int knet_handle_setprio_dscp(knet_handle_t knet_h, uint8_t dscp);
```

Comment:
each new API call must have a dedicated test in libknet/tests/ and coordinate with @chrissie-c for rust binding support